aws/config.go

```diff
@@ -69,6 +69,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/iam"
 	"github.com/aws/aws-sdk-go/service/inspector"
 	"github.com/aws/aws-sdk-go/service/iot"
+	"github.com/aws/aws-sdk-go/service/kafka"
```

Comment:
I believe the convention is to use the official AWS service name, so would be msk instead of Kafka here and further along in the PR.

Reply:
In the AWS API it looks like it is referred to as `Kafka` instead of `msk`
https://docs.aws.amazon.com/sdk-for-go/api/service/kafka/